ocs_ci/ocs/amq.py

```diff
@@ -211,12 +480,21 @@ def cleanup(self):
         then amq-connector, persistent, bridge, at the end it will delete the created namespace
         """
         if self.amq_is_setup:
+            if self.messaging:
+                self.consumer_pod.delete()
+                self.producer_pod.delete()
+                self.kafka_user.delete()
+                self.kafka_topic.delete()
             self.kafka_persistent.delete()
             self.kafka_connect.delete()
             self.kafka_bridge.delete()
             run_cmd(f'oc delete -f {self.amq_dir}', shell=True, check=True, cwd=self.dir)
-            run_cmd(f'oc delete -f {self.amq_dir_examples}', shell=True, check=True, cwd=self.dir)
         run_cmd(f'oc delete project {self.namespace}')
+
+        # Change the existing default Storageclass annotation to false
```

Comment:
You are setting the current storage class as false. 
We have to set the previous sc as default storage class else we may end up with no default sc in the cluster.